### PR TITLE
Simplify FXML files #231

### LINKED ISSUE #231: Simplify FXML files
As noted in https://github.com/se-edu/addressbook-level4/pull/200#discussion_r91964165 the FXML files could probably be simplified.

Simplifications could include:

* Removing unnecessary attributes (the defaults already give the same value/behavior)
* Removing unnecessary nesting of Panes (HBox, VBox, AnchorPanes...)

There should be no behavioral or visual changes.

### PR DESCRIPTION
**Pull request by MightyCupcakes**
Closes #231 

Propsed merge commit message:

```
The FXML files are deeply nested with unnecessary parent elements.
There are also multiple conflicting coding styles that exists within
the same file.

This makes the files hard to read and navigate. Furthermore, the
differing coding styles in the same file makes the whole file
inconsistent.

Let's modify the FXML files as follows:
    * Change indentations to 2 spaces in all FXML files as these files
      tends to have deep nesting
    * Remove all star imports and reorder the rest in alphabetical
      order in all FXML files
    * Remove unnecessary parent elements to reduce the nesting in
      all FXML files
    * Remove all unnecessary CSS style sheet imports except those
      in MainWindow.fxml
    * Wrap long attributes into multiple lines and add white spaces
      in MainWindow.fxml to improve readability
    * Remove unneeded minwidth attribute in StatusBarFooter.fxml
    * Remove unnecessary HBox element in PersonListCard.fxml
    * Standardize inset attributes order declarations in all FXML files
    * Add a space before closing slash in self-closing tags in all FXML files
    * Standardize height and width declarations in all FXML files to use
      only integer values 

```

### CODE REVIEW

**Comment by CanIHasReview** [comment]
# v1

@MightyCupcakes submitted v1 for review.

* [v1 1/6] [Change indentation to 2 spaces in all FXML files](https://github.com/se-edu/addressbook-level4/pull/356/commits/a649e2a2a520dffc7d42de17f87b82db3de95910)
* [v1 2/6] [Refactor imports in all FXML files](https://github.com/se-edu/addressbook-level4/pull/356/commits/500f0a53fe711430d3acd3bed76621630474f301)
* [v1 3/6] [Remove unneeded parent elements](https://github.com/se-edu/addressbook-level4/pull/356/commits/fe6e2e8ab85bfe39d30f62d356e74dfa5d9bf246)
* [v1 4/6] [PersonListCard\.fxml: Refactor and remove unnecessary nesting](https://github.com/se-edu/addressbook-level4/pull/356/commits/8aad703e85f6250d110457ea8b6d22141f79bb21)
* [v1 5/6] [MainWindow\.fxml: Refactor and remove unnecessary nesting](https://github.com/se-edu/addressbook-level4/pull/356/commits/cea7b69ae5f8e2edb82f7c49fcc260955905ae6b)
* [v1 6/6] [Remove stylesheets imports in UiPart's FXML files](https://github.com/se-edu/addressbook-level4/pull/356/commits/f8865c61129dca20fd7740f033c68ce49cf8c5cd)

([:books: Archive](https://github.com/se-edu/addressbook-level4/compare/e1eafd1145cb5fdcc81bbacecf9e5145903b3fe2...f8865c61129dca20fd7740f033c68ce49cf8c5cd))
<details>
<summary>Checkout this PR version locally</summary>

```shell
git fetch https://github.com/se-edu/addressbook-level4.git refs/pr/356/1/head:BRANCHNAME
```
where `BRANCHNAME` is the name of the local branch you want to fetch this PR to.

</details>

File: src/main/resources/view/StatusBarFooter.fxml [hunk 1]
```diff
@@ -9,9 +9,7 @@
     <ColumnConstraints hgrow="SOMETIMES" minWidth="10.0" prefWidth="100.0" />
     <ColumnConstraints hgrow="SOMETIMES" minWidth="10.0" prefWidth="100.0" />
   </columnConstraints>
-  <children>
-    <StatusBar styleClass="anchor-pane" fx:id="syncStatus" minWidth="0.0"/>
-    <StatusBar styleClass="anchor-pane" fx:id="saveLocationStatus" minWidth="0.0" GridPane.columnIndex="1"
-      nodeOrientation="RIGHT_TO_LEFT" />
-  </children>
+
+  <StatusBar styleClass="anchor-pane" fx:id="syncStatus" />
```
Comment (chao1995) [inline src/main/resources/view/StatusBarFooter.fxml]: minWidth="0.0" is removed, but seems like it is not related to this commit.

File: src/main/resources/view/StatusBarFooter.fxml [hunk 2]
```diff
@@ -9,9 +9,7 @@
...
+  <StatusBar styleClass="anchor-pane" fx:id="saveLocationStatus" GridPane.columnIndex="1" nodeOrientation="RIGHT_TO_LEFT" />
```
Comment (chao1995) [inline src/main/resources/view/StatusBarFooter.fxml]: `minWidth="0.0"` is removed, but seems like it is not related to this commit.

Comment (chao1995) [inline src/main/resources/view/StatusBarFooter.fxml]: minWidth="0.0" is removed, but seems like it is not related to this commit.

File: src/main/resources/view/ResultDisplay.fxml [hunk 3]
```diff
@@ -1,15 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<?import java.net.URL?>
```
Comment (chao1995) [inline src/main/resources/view/ResultDisplay.fxml]: Should this removal be done in [v1 2/6] or is this caused by removing stylesheet tag? As I saw `personlistcard.fxml" actually gets rid of this import before removing the stylesheet tag in [v1 5/6]

File: src/main/resources/view/MainWindow.fxml [hunk 4]
```diff
@@ -14,48 +14,45 @@
...
-    </AnchorPane>
-    <AnchorPane VBox.vgrow="NEVER" fx:id="resultDisplayPlaceholder" styleClass="anchor-pane-with-border" minHeight="100" prefHeight="100" maxHeight="100">
+
+      <AnchorPane fx:id="personListPanelPlaceholder" VBox.vgrow="ALWAYS" />
+    </VBox>
+
+    <AnchorPane fx:id="browserPlaceholder" prefWidth="340" >
       <padding>
-        <Insets top="5.0" bottom="5.0" left="10.0" right="10.0"/>
+        <Insets top="10" bottom="10" left="10" right="10" />
```
Comment (PierceAndy) [inline src/main/resources/view/MainWindow.fxml]: Change unrelated to commit.

Comment (PierceAndy) [inline src/main/resources/view/MainWindow.fxml]: Change unrelated to commit.

**Comment by CanIHasReview** [comment]
# v2

@MightyCupcakes submitted v2 for review.

* [v2 1/7] [Change indentation to 2 spaces in all FXML files](https://github.com/se-edu/addressbook-level4/pull/356/commits/9da3a20a08fa299a882206a3b83ee470e10cb46e)
* [v2 2/7] [Refactor imports in all FXML files](https://github.com/se-edu/addressbook-level4/pull/356/commits/199ed299d5692ed3fe14e66c937289f55c7710e4)
* [v2 3/7] [Remove unnecessary children tags in FXML files](https://github.com/se-edu/addressbook-level4/pull/356/commits/25d23948d0987837fb91017d131a45356df88575)
* [v2 4/7] [Remove stylesheets imports in UiPart's FXML files](https://github.com/se-edu/addressbook-level4/pull/356/commits/a074074351d5e0c5352ceccd3f4155488ea5d090)
* [v2 5/7] [MainWindow\.fxml: Refactor to improve readability](https://github.com/se-edu/addressbook-level4/pull/356/commits/c99f60acf1bfbec2f67a2326595eceeb541ffe4e)
* [v2 6/7] [StatusBarFooter\.fxml: Remove unneeded attribute](https://github.com/se-edu/addressbook-level4/pull/356/commits/0e6f1b5c673e4165bccdeedfc425da665179b7a4)
* [v2 7/7] [PersonListCard\.fxml: Remove unneeded child HBox](https://github.com/se-edu/addressbook-level4/pull/356/commits/ffd64bd8c142cdab95b41f9a0f6e12ea00246d60)

([:books: Archive](https://github.com/se-edu/addressbook-level4/compare/e1eafd1145cb5fdcc81bbacecf9e5145903b3fe2...ffd64bd8c142cdab95b41f9a0f6e12ea00246d60)) ([:chart_with_upwards_trend: Interdiff between v1 and v2](https://github.com/se-edu/addressbook-level4/compare/4f223b491575f486f6803c0be49c648f29adde9a~1...4f223b491575f486f6803c0be49c648f29adde9a))
<details>
<summary>Checkout this PR version locally</summary>

```shell
git fetch https://github.com/se-edu/addressbook-level4.git refs/pr/356/2/head:BRANCHNAME
```
where `BRANCHNAME` is the name of the local branch you want to fetch this PR to.

</details>

**Comment by MightyCupcakes** [comment]
# V2

Thanks @chao1995 and @PierceAndy 

The commit organisation is a bit off last time.

Reorganised commits according to reviews.

**Comment by MightyCupcakes** [comment]
@chao1995 

My bad, I think the better word is load.

To be more specific, `MainWindow` loads all UiParts by adding their various components as a child of the `AnchorPane` placeholders. Since the `AnchorPane` placeholder is part of MainWindow, all its children will also be styled by the stylesheets declared in MainWindow.

**Comment by chao1995** [comment]
@MightyCupcakes 

Thanks for the explanation. So I guess you can mention (in the commit message)

>MainWindow loads all UiParts by adding their various components as a child of the AnchorPane placeholders. Since the AnchorPane placeholder is part of MainWindow, all its children will also be styled by the stylesheets declared in MainWindow.

And thus it's unnecessary to redeclare the imports in each `UiPart.fxml` unless it needs further customisation.

**Comment by CanIHasReview** [comment]
# v3

@MightyCupcakes submitted v3 for review.

* [v3 1/7] [Change indentation to 2 spaces in all FXML files](https://github.com/se-edu/addressbook-level4/pull/356/commits/dab7558926865458bf3ff8d5d477dbbc78066a48)
* [v3 2/7] [Refactor imports in all FXML files](https://github.com/se-edu/addressbook-level4/pull/356/commits/9023aec2f8e453d004b731e2b37380270f241138)
* [v3 3/7] [Remove unnecessary children tags in FXML files](https://github.com/se-edu/addressbook-level4/pull/356/commits/91df09266f12de1d5a307421f85597be9664e19f)
* [v3 4/7] [Remove stylesheets imports in UiPart's FXML files](https://github.com/se-edu/addressbook-level4/pull/356/commits/9e4a3d50f1a1cdbb24178965b50bd45912ecbb32)
* [v3 5/7] [MainWindow\.fxml: Refactor to improve readability](https://github.com/se-edu/addressbook-level4/pull/356/commits/eb8a2b5d9531f5b86b12eb6b11b1d8f5af511ee8)
* [v3 6/7] [StatusBarFooter\.fxml: Remove unneeded attribute](https://github.com/se-edu/addressbook-level4/pull/356/commits/60a55a9196c1358f7970026effa3c5bedcabe047)
* [v3 7/7] [PersonListCard\.fxml: Remove unneeded child HBox](https://github.com/se-edu/addressbook-level4/pull/356/commits/98bc22d7c4c2f7c7f48efaf8df37f49fd1e15cad)

([:books: Archive](https://github.com/se-edu/addressbook-level4/compare/27cb4621dca1bf474b2838352a90873f3fbcb4ef...98bc22d7c4c2f7c7f48efaf8df37f49fd1e15cad)) ([:chart_with_upwards_trend: Interdiff between v2 and v3](https://github.com/se-edu/addressbook-level4/compare/da828b30a1c8104825db86b0a287d5464a7ab792~1...da828b30a1c8104825db86b0a287d5464a7ab792))
<details>
<summary>Checkout this PR version locally</summary>

```shell
git fetch https://github.com/se-edu/addressbook-level4.git refs/pr/356/3/head:BRANCHNAME
```
where `BRANCHNAME` is the name of the local branch you want to fetch this PR to.

</details>

File: src/main/resources/view/MainWindow.fxml [hunk 5]
```diff
@@ -14,48 +14,36 @@
...
+  <AnchorPane VBox.vgrow="NEVER" fx:id="resultDisplayPlaceholder" styleClass="anchor-pane-with-border" minHeight="100" prefHeight="100" maxHeight="100">
+    <padding>
+      <Insets top="5.0" bottom="5.0" left="10.0" right="10.0"/>
+    </padding>
+  </AnchorPane>
+  <SplitPane id="splitPane" fx:id="splitPane" dividerPositions="0.4" VBox.vgrow="ALWAYS">
+    <VBox fx:id="personList" minWidth="340" prefWidth="340">
       <padding>
-        <Insets top="5.0" bottom="5.0" left="10.0" right="10.0"/>
+        <Insets bottom="10.0" left="10.0" right="10.0" top="10.0"/>
```
Comment (PierceAndy) [inline src/main/resources/view/MainWindow.fxml]: Change unrelated to `[v3 3/7] Remove unnecessary children tags in FXML files`?

File: src/main/resources/view/MainWindow.fxml [hunk 6]
```diff
@@ -14,48 +14,36 @@
...
       </padding>
-    </AnchorPane>
-    <AnchorPane VBox.vgrow="NEVER" fx:id="resultDisplayPlaceholder" styleClass="anchor-pane-with-border" minHeight="100" prefHeight="100" maxHeight="100">
+      <AnchorPane fx:id="personListPanelPlaceholder" VBox.vgrow="ALWAYS"/>
+    </VBox>
+    <AnchorPane fx:id="browserPlaceholder" prefWidth="340" >
       <padding>
-        <Insets top="5.0" bottom="5.0" left="10.0" right="10.0"/>
+        <Insets top="10" bottom="10" left="10" right="10"/>
```
Comment (PierceAndy) [inline src/main/resources/view/MainWindow.fxml]: Change unrelated to `[v3 3/7] Remove unnecessary children tags in FXML files`?

File: src/main/resources/view/MainWindow.fxml (hunk 5 above)
Comment (acjh) [inline src/main/resources/view/MainWindow.fxml]: Consider keeping the order (top, bottom, left, right) since this is not the only case in this file?

File: src/main/resources/view/MainWindow.fxml (hunk 6 above)
Comment (MightyCupcakes) [inline src/main/resources/view/MainWindow.fxml]: as mentioned above, the diff associated similar lines moved up due to the deletion of parent elements to the original file. If you really compare the two files, you can see nothing has changed.

File: src/main/resources/view/MainWindow.fxml (hunk 5 above)
Comment (MightyCupcakes) [inline src/main/resources/view/MainWindow.fxml]: I did not change anything. The diff wrongly associated the line moved up (due to deletion of parent elements above) in this commit with another line in the original file.

Comment (PierceAndy) [inline src/main/resources/view/MainWindow.fxml]: Ah yes you're right. Sorry, the links provided in your v2 message don't work any more, so I didn't realize they pointed to these lines.

Comment (PierceAndy) [inline src/main/resources/view/MainWindow.fxml]: @acjh Has a good point about the order being inconsistent though. Maybe one more commit to standardize the order across all FXML files?

Comment (acjh) [inline src/main/resources/view/MainWindow.fxml]: Understood. How about adding a commit for stylistic changes (standardizing the order and use of `.0` or not) since we are already modifying these lines?

Comment (acjh) [inline src/main/resources/view/MainWindow.fxml]: Beat me to it lol

Comment (MightyCupcakes) [inline src/main/resources/view/MainWindow.fxml]: Good point. But there is currently no standard to follow in this case.

Comment (acjh) [inline src/main/resources/view/MainWindow.fxml]: How about top-right-bottom-left to mirror [its constructor](https://docs.oracle.com/javase/8/javafx/api/javafx/geometry/Insets.html)?

Comment (acjh) [inline src/main/resources/view/MainWindow.fxml]: A lesser but consistent rationale: it's also the [convention for CSS](https://www.w3schools.com/css/css_margin.asp)!

Comment (acjh) [inline src/main/resources/view/MainWindow.fxml]: As for `.0`, I'm *for* leaving them out. See [this related SO post](http://stackoverflow.com/questions/30447692/decimal-values-for-paddings-in-javafx-are-rounded-to-integer-why).

**Comment by CanIHasReview** [comment]
# v4

@MightyCupcakes submitted v4 for review.

* [v4 1/8] [Change indentation to 2 spaces in all FXML files](https://github.com/se-edu/addressbook-level4/pull/356/commits/dab7558926865458bf3ff8d5d477dbbc78066a48)
* [v4 2/8] [Refactor imports in all FXML files](https://github.com/se-edu/addressbook-level4/pull/356/commits/9023aec2f8e453d004b731e2b37380270f241138)
* [v4 3/8] [Remove unnecessary children tags in FXML files](https://github.com/se-edu/addressbook-level4/pull/356/commits/91df09266f12de1d5a307421f85597be9664e19f)
* [v4 4/8] [Remove stylesheets imports in UiPart's FXML files](https://github.com/se-edu/addressbook-level4/pull/356/commits/9e4a3d50f1a1cdbb24178965b50bd45912ecbb32)
* [v4 5/8] [MainWindow\.fxml: Refactor to improve readability](https://github.com/se-edu/addressbook-level4/pull/356/commits/eb8a2b5d9531f5b86b12eb6b11b1d8f5af511ee8)
* [v4 6/8] [StatusBarFooter\.fxml: Remove unneeded attribute](https://github.com/se-edu/addressbook-level4/pull/356/commits/60a55a9196c1358f7970026effa3c5bedcabe047)
* [v4 7/8] [PersonListCard\.fxml: Remove unneeded child HBox](https://github.com/se-edu/addressbook-level4/pull/356/commits/98bc22d7c4c2f7c7f48efaf8df37f49fd1e15cad)
* [v4 8/8] [Standardize declaration order of insets in all FXML files](https://github.com/se-edu/addressbook-level4/pull/356/commits/12a8272acb16090c43af6636540f28b60314a15b)

([:books: Archive](https://github.com/se-edu/addressbook-level4/compare/27cb4621dca1bf474b2838352a90873f3fbcb4ef...12a8272acb16090c43af6636540f28b60314a15b)) ([:chart_with_upwards_trend: Interdiff between v3 and v4](https://github.com/se-edu/addressbook-level4/compare/5d174f09b8dc71407b15fc07d66fea3726483666~1...5d174f09b8dc71407b15fc07d66fea3726483666))
<details>
<summary>Checkout this PR version locally</summary>

```shell
git fetch https://github.com/se-edu/addressbook-level4.git refs/pr/356/4/head:BRANCHNAME
```
where `BRANCHNAME` is the name of the local branch you want to fetch this PR to.

</details>

File: src/main/resources/view/PersonListCard.fxml [hunk 7]
```diff
@@ -1,53 +1,36 @@
...
-                        <Label fx:id="address" styleClass="cell_small_label" text="\$address" />
-                        <Label fx:id="email" styleClass="cell_small_label" text="\$email" />
-                    </children>
-                </VBox>
-            </children>
-        </GridPane>
-    </children>
+  <GridPane HBox.hgrow="ALWAYS">
+    <columnConstraints>
+      <ColumnConstraints hgrow="SOMETIMES" minWidth="10.0" prefWidth="150.0" />
```
Comment (acjh) [inline src/main/resources/view/PersonListCard.fxml]: Standardize no `.0` for width and height as well?

File: src/main/resources/view/BrowserPanel.fxml [hunk 8]
```diff
@@ -4,5 +4,5 @@
 <?import javafx.scene.web.WebView?>
 
 <AnchorPane xmlns:fx="http://javafx.com/fxml/1">
-    <WebView fx:id="browser"/>
+  <WebView fx:id="browser"/>
```
Comment (acjh) [inline src/main/resources/view/BrowserPanel.fxml]: Standardize with space in `" />` for self-closing tags?

**Comment by CanIHasReview** [comment]
# v5

@MightyCupcakes submitted v5 for review.

* [v5 1/10] [Change indentation to 2 spaces in all FXML files](https://github.com/se-edu/addressbook-level4/pull/356/commits/dab7558926865458bf3ff8d5d477dbbc78066a48)
* [v5 2/10] [Refactor imports in all FXML files](https://github.com/se-edu/addressbook-level4/pull/356/commits/9023aec2f8e453d004b731e2b37380270f241138)
* [v5 3/10] [Remove unnecessary children tags in FXML files](https://github.com/se-edu/addressbook-level4/pull/356/commits/91df09266f12de1d5a307421f85597be9664e19f)
* [v5 4/10] [Remove stylesheets imports in UiPart's FXML files](https://github.com/se-edu/addressbook-level4/pull/356/commits/9e4a3d50f1a1cdbb24178965b50bd45912ecbb32)
* [v5 5/10] [MainWindow\.fxml: Refactor to improve readability](https://github.com/se-edu/addressbook-level4/pull/356/commits/eb8a2b5d9531f5b86b12eb6b11b1d8f5af511ee8)
* [v5 6/10] [StatusBarFooter\.fxml: Remove unneeded attribute](https://github.com/se-edu/addressbook-level4/pull/356/commits/60a55a9196c1358f7970026effa3c5bedcabe047)
* [v5 7/10] [PersonListCard\.fxml: Remove unneeded child HBox](https://github.com/se-edu/addressbook-level4/pull/356/commits/98bc22d7c4c2f7c7f48efaf8df37f49fd1e15cad)
* [v5 8/10] [Standardize declaration order of insets in all FXML files](https://github.com/se-edu/addressbook-level4/pull/356/commits/55c64d34e21dd43d5604aa8543adf254a944ad98)
* [v5 9/10] [Standardize self closing tags in all FXML files](https://github.com/se-edu/addressbook-level4/pull/356/commits/2ca475d46a8b71274e64fe3708aa4270d01b6976)
* [v5 10/10] [Standardize height and width declarations in all FXML files](https://github.com/se-edu/addressbook-level4/pull/356/commits/4b016f605104f082ad45dc4a8e900576fc9d6c5f)

([:books: Archive](https://github.com/se-edu/addressbook-level4/compare/27cb4621dca1bf474b2838352a90873f3fbcb4ef...4b016f605104f082ad45dc4a8e900576fc9d6c5f)) ([:chart_with_upwards_trend: Interdiff between v4 and v5](https://github.com/se-edu/addressbook-level4/compare/2a4601a26246f05c06938ce8aabeb90a0861023f~1...2a4601a26246f05c06938ce8aabeb90a0861023f))
<details>
<summary>Checkout this PR version locally</summary>

```shell
git fetch https://github.com/se-edu/addressbook-level4.git refs/pr/356/5/head:BRANCHNAME
```
where `BRANCHNAME` is the name of the local branch you want to fetch this PR to.

</details>

**Comment by PierceAndy** [comment]
@chao1995 One last check?

**Comment by MightyCupcakes** [comment]
The space is meaningless and its mainly for readability. However, most existing self-closing tags in the FXML files have a space and only the odd few did not leave the space, making it very inconsistent. We should decide on one and stick to it.

**Comment by PierceAndy** [comment]
I don't think having or not having the space makes a large difference in this case. It's just a matter of choosing a standard and sticking with it.

We can use this chance to define a coding standard for XML files here: https://github.com/oss-generic/process/issues/84

**Comment by MightyCupcakes** [comment]
As discussed today, it is decided that the standard should be leaving a space before the closing slash in all self closing tags.

**Comment by damithc** [comment]
@yl-coder any inputs?

**Comment by damithc** [comment]
@MightyCupcakes can you propose a merge commit message too?

File: src/main/resources/view/MainWindow.fxml [hunk 9]
```diff
@@ -22,28 +23,34 @@
...
+
   <AnchorPane VBox.vgrow="NEVER" fx:id="commandBoxPlaceholder" styleClass="anchor-pane-with-border">
     <padding>
       <Insets top="5.0" bottom="5.0" left="10.0" right="10.0"/>
     </padding>
   </AnchorPane>
-  <AnchorPane VBox.vgrow="NEVER" fx:id="resultDisplayPlaceholder" styleClass="anchor-pane-with-border" minHeight="100" prefHeight="100" maxHeight="100">
+
+  <AnchorPane VBox.vgrow="NEVER" fx:id="resultDisplayPlaceholder" styleClass="anchor-pane-with-border"
+    minHeight="100" prefHeight="100" maxHeight="100">
```
Comment (damithc) [inline src/main/resources/view/MainWindow.fxml]: When line wrapping, indentation should be twice the normal? e.g. Java uses 8 spaces when line wrapping.

File: src/main/resources/view/CommandBox.fxml [hunk 10]
```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
```
Comment (damithc) [inline src/main/resources/view/CommandBox.fxml]: Commit 2 message:

>Let's refactor the imports in all FXML files to follow the project's
coding standards.

This statement promises too much. How about this?

```
Let's refactor ... to fix the two problems given above.
```

File: src/main/resources/view/StatusBarFooter.fxml [hunk 11]
```diff
@@ -9,7 +9,6 @@
     <ColumnConstraints hgrow="SOMETIMES" minWidth="10.0" prefWidth="100.0" />
     <ColumnConstraints hgrow="SOMETIMES" minWidth="10.0" prefWidth="100.0" />
   </columnConstraints>
-  <StatusBar styleClass="anchor-pane" fx:id="syncStatus" minWidth="0.0"/>
```
Comment (damithc) [inline src/main/resources/view/StatusBarFooter.fxml]: Commit 6 message:
`Both StatusBar` -> `Both StatusBar elements`?
`both status bar will` > `both status bars will`

**Comment by CanIHasReview** [comment]
# v6

@MightyCupcakes submitted v6 for review.

* [v6 1/10] [Change indentation to 2 spaces in all FXML files](https://github.com/se-edu/addressbook-level4/pull/356/commits/f508198e1b0fd1c7ba7e6afa57cf7d68ba033a48)
* [v6 2/10] [Refactor imports in all FXML files](https://github.com/se-edu/addressbook-level4/pull/356/commits/0f1ddead901d40d905f0cc87c910299bfd29616a)
* [v6 3/10] [Remove unnecessary children tags in FXML files](https://github.com/se-edu/addressbook-level4/pull/356/commits/83b1c167f18613cff22a20556b61250094ca4515)
* [v6 4/10] [Remove stylesheets imports in UiPart's FXML files](https://github.com/se-edu/addressbook-level4/pull/356/commits/75ccaf2b9dc28d254aecdb06c7c2a52cc08a852e)
* [v6 5/10] [MainWindow\.fxml: Refactor to improve readability](https://github.com/se-edu/addressbook-level4/pull/356/commits/fbced2b602d7c537d9b7372f271527c8b72504f2)
* [v6 6/10] [StatusBarFooter\.fxml: Remove unneeded attribute](https://github.com/se-edu/addressbook-level4/pull/356/commits/3980de399354dfb89134d615f896d8831eb36d96)
* [v6 7/10] [PersonListCard\.fxml: Remove unneeded child HBox](https://github.com/se-edu/addressbook-level4/pull/356/commits/c071a8a005d4ba9c67fbd7478837a06fa030141c)
* [v6 8/10] [Standardize declaration order of insets in all FXML files](https://github.com/se-edu/addressbook-level4/pull/356/commits/0434e044c5dc28d9f05bb5fcd947611b4ad9725a)
* [v6 9/10] [Standardize self closing tags in all FXML files](https://github.com/se-edu/addressbook-level4/pull/356/commits/8c02003ab6f2fdc6ff0e0fdf4842fd811a7f5a44)
* [v6 10/10] [Standardize height and width declarations in all FXML files](https://github.com/se-edu/addressbook-level4/pull/356/commits/8380bc898f4d9d408e25c959354245086d8c74f1)

([:books: Archive](https://github.com/se-edu/addressbook-level4/compare/07367fab5054768f3243fb55eba1367d3d67a39b...8380bc898f4d9d408e25c959354245086d8c74f1)) ([:chart_with_upwards_trend: Interdiff between v5 and v6](https://github.com/se-edu/addressbook-level4/compare/be27a54984e9c1d277b07b760ee72a14cb7e0ba3~1...be27a54984e9c1d277b07b760ee72a14cb7e0ba3))
<details>
<summary>Checkout this PR version locally</summary>

```shell
git fetch https://github.com/se-edu/addressbook-level4.git refs/pr/356/6/head:BRANCHNAME
```
where `BRANCHNAME` is the name of the local branch you want to fetch this PR to.

</details>

File: src/main/resources/view/ResultDisplay.fxml [hunk 12]
```diff
@@ -1,15 +1,11 @@
...
 <AnchorPane fx:id="placeHolder" styleClass="anchor-pane-with-border" xmlns="http://javafx.com/javafx/8"
-            xmlns:fx="http://javafx.com/fxml/1">
-    <stylesheets>
-        <URL value="@DarkTheme.css"/>
-        <URL value="@Extensions.css"/>
-    </stylesheets>
-    <AnchorPane fx:id="mainPane">
-        <TextArea fx:id="resultDisplay" editable="false" styleClass="result-display"/>
-    </AnchorPane>
+  xmlns:fx="http://javafx.com/fxml/1">
```
Comment (acjh) [inline src/main/resources/view/ResultDisplay.fxml]: Double indentation for line wrapping here too

**Comment by CanIHasReview** [comment]
# v7

@MightyCupcakes submitted v7 for review.

* [v7 1/10] [Change indentation to 2 spaces in all FXML files](https://github.com/se-edu/addressbook-level4/pull/356/commits/5bc7b615fe1fbde08c06822bcf439a864b124311)
* [v7 2/10] [Refactor imports in all FXML files](https://github.com/se-edu/addressbook-level4/pull/356/commits/f8ae1f0efc2e6724be643b3c003d6e6bd3bd8e10)
* [v7 3/10] [Remove unnecessary children tags in FXML files](https://github.com/se-edu/addressbook-level4/pull/356/commits/895f252667f346a88db65a7d5fb0e22c9d138f25)
* [v7 4/10] [Remove stylesheets imports in UiPart's FXML files](https://github.com/se-edu/addressbook-level4/pull/356/commits/38a3cb6997dff47e8a6280405f9cc0539e7ffa75)
* [v7 5/10] [MainWindow\.fxml: Refactor to improve readability](https://github.com/se-edu/addressbook-level4/pull/356/commits/1919a0ab3a347e3a825ac4a83680d51edf767f8b)
* [v7 6/10] [StatusBarFooter\.fxml: Remove unneeded attribute](https://github.com/se-edu/addressbook-level4/pull/356/commits/3c21984b8a7cf254ed29b3a76794310502a78a4b)
* [v7 7/10] [PersonListCard\.fxml: Remove unneeded child HBox](https://github.com/se-edu/addressbook-level4/pull/356/commits/098c151f88a6538e5f41e3301e9ee9ede82c9524)
* [v7 8/10] [Standardize declaration order of insets in all FXML files](https://github.com/se-edu/addressbook-level4/pull/356/commits/8f4b7a06d8d784b64246fd03f4d1eb23fced8e5e)
* [v7 9/10] [Standardize self closing tags in all FXML files](https://github.com/se-edu/addressbook-level4/pull/356/commits/1796655b975ea47319dbe96f10661b720dd5a99c)
* [v7 10/10] [Standardize height and width declarations in all FXML files](https://github.com/se-edu/addressbook-level4/pull/356/commits/b91565e39e0a18b3f5634912c1849a6aa599f4a6)

([:books: Archive](https://github.com/se-edu/addressbook-level4/compare/07367fab5054768f3243fb55eba1367d3d67a39b...b91565e39e0a18b3f5634912c1849a6aa599f4a6)) ([:chart_with_upwards_trend: Interdiff between v6 and v7](https://github.com/se-edu/addressbook-level4/compare/1e00963b79aa98509bf95782fef696075a17a042~1...1e00963b79aa98509bf95782fef696075a17a042))
<details>
<summary>Checkout this PR version locally</summary>

```shell
git fetch https://github.com/se-edu/addressbook-level4.git refs/pr/356/7/head:BRANCHNAME
```
where `BRANCHNAME` is the name of the local branch you want to fetch this PR to.

</details>

**Comment by damithc** [comment]
@MightyCupcakes 

Regarding the proposed commit message:
* This statement is too general `Re-factor the codes to follow a consistent coding style`
* Given the nature of this PR, may be the merge commit message can be an itemized list of the 10 refactorings done, somewhat similar to the CIHR commit summary, but with a tiny bit more details.

**Comment by MightyCupcakes** [comment]
@damithc update proposed merge commit message
